### ISSUE TITLE
Feature request or wish list

### ISSUE THREAD

**Issue by mitjakac** [LikaloLLC/docsie#12]
Hello,
I'm playing with Docsie and very satisfied. Thank you for your effort.

Few questions or wish list:

- Is there possibility to link to specific section of documentation? So I would have different link on different sections of my application. Something like 
src="https://lib.docsie.io/1.1.0/service.js" data-docsie="test_pk_xxx, section:**_my section name_**

- Do you have plans to allow Export to PDF? For using offline.

- Also, is it possible to export documentation and include locally for application behind firewall?
So, it would be src="http:/localhost/service.js" with all data inside this or separate .js file

**Comment by PhilippeTrounev** [comment]
Hi Mitjakac,
 Yes these features are currently in works. PDF export should be available soon. Offline packaging is on the way as well. 

In regards to linking different sections, right now it's implemented on the url level for example: https://www.docsie.io/documentation/?version=0.0.1&language=EN&article=versions#section-header-two-crhkd 

Can you elaborate a bit more on this?

**Comment by mitjakac** [comment]
Hello Philippe,
thank you for your answer.
Looking forward for this new features.

Regarding linking, not sure I understood you (or you me ) :-(
Sorry if this is stupid question.

My current setup for testing:
- on my index page have script tag for https://lib.docsie.io/1.1.0/service.js and data-docsie tag with my id
- hidden DIV with data-ddsroot tag
- Help link which shows hidden div with full documentation

I would like to have separate help links on every page that links specific article / section.
How can I call docsie docummentation of specific section and load to some help div?

Thank you and best regards

**Comment by PhilippeTrounev** [comment]
Yes this makes sense. There is a feature in works to allow that. It should be coming in two more sprints, so most likely in February. This feature would allow you to specify render of specific article in the script. I will update the story to allow it to optionally pass section so it could render the correct article/section in the scope of a div.
